msteams-platform/samples/app-templates.md

```diff
@@ -210,6 +210,20 @@ E-Prescriptions is a [Power Apps](/powerapps/maker/canvas-apps/embed-teams-app)-
 :::column-end:::
 :::row-end:::
 
+
+## Emergency Button App 
+
+The Emergency Button Power App can be used by organisations who are using Microsoft Teams to enable any set of users to request assistance from supervisors. Features include:
+
+-   Requesting assistance on different categories from a Power App
+-   Notifications sent to requestor informing them of who has been assigned 
+-   Notifications sent to assigned supervisor informing them of who needs assistance 
+-   Audit trail held in SharePoint site
+-   And many more.
+
+[Get it on GitHub](https://github.com/OfficeDev/microsoft-teams-apps-emergency-button-app)
```

Comment:
The link is incorrect, please verify.

Reply:
It was private until approved, changed to public now